### PR TITLE
add show detail course

### PR DESCRIPTION
**Pull request by ducth-2598**
image
Screenshot from 2021-07-29 12-23-19

redmine: https://edu-redmine.sun-asterisk.vn/issues/38984

### CODE REVIEW

File: training_system/app/views/trainer/courses/show.js.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$('.modal-body').html('<%= escape_javascript(render :partial => 'trainee_of_course', collection: @negotiatitraineeson, as: :trainee) %>');
+$('.modal-header').remove();
+
```
Comment (thiephn-2606-zz) [inline training_system/app/views/trainer/courses/show.js.erb]: xóa line man

File: training_system/app/models/course.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Course < ApplicationRecord
   has_many :user_courses, dependent: :destroy
   has_many :users, through: :user_courses
-  has_many :subjects, through: :course_subjects
   has_many :course_subjects, dependent: :destroy
+  has_many :subjects, through: :course_subjects
```
Comment (thiephn-2606) [inline training_system/app/models/course.rb:5]: sữa lại như cũ fen

**Comment by thiephn-2606-zz** [comment]
approve

**Comment by ducth-2598** [comment]
ready

**Pull request merged**